src/changelog-updater.ts

```diff
@@ -224,7 +224,7 @@ async function parseChangelogForEntry(
           dependencySectionFound = DEPENDENCY_SECTION_REGEX.test(line)
           changelogLineNumber = lineNumber + 1
         }
-      } else if (line.startsWith('- ')) {
+      } else if (line.trim().startsWith('- ')) {
```

Comment:
I do wonder if we should just trim the line at the outset and use that throughout.

Reply:
That would mean that also lines starting with `" ### "` would be considered valid, do you want that?

Also the line variable would need to be made mutable, or a new `const trimmedLine` would need to be introduced.

Maybe it would be better to get rid of the trim and replace the startsWith with a regular expression (`/^\s*- /`)?

Reply:
Good points.

I'm fine with using another regular expression. It feels more appropriate the more I think about it